src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,100 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int COEFFICIENT_GROW = 2;
+    private static final int DEFAULT_CAPACITY = 16;
+    private Node<K, V>[] map;
+    private int capacity;
+    private int size;
+
+    public MyHashMap() {
+        capacity = DEFAULT_CAPACITY;
+        map = new Node[capacity];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size + 1 >= capacity * LOAD_FACTOR) {
+            resizeMap();
+        }
+        putItem(map, capacity, key, value);
```

Comment:
```suggestion
        putItem(map, key, value);
```
take capacity inside `putItem()` method from `map.length`

Reply:
ok